src/provider.ts

```diff
@@ -158,3 +167,16 @@ export class DelphinusWalletConnector extends DelphinusSigner<Wallet> {
     return await this.signer.call(req);
   }
 }
+
+export interface AddNetworkOptions {
+  chainId: string;
```

Comment:
please as some example comments to show how to fill it, for example, this chainId should be // Should be HexNumber string. Example: "0x##"